src/library/dao/trackdao.cpp

```diff
@@ -289,14 +289,10 @@ QString TrackDAO::getTrackLocation(TrackId trackId) const {
 
 void TrackDAO::saveTrack(Track* pTrack) const {
     DEBUG_ASSERT(pTrack);
```

Comment:
```suggestion
    VERIFY_OR_DEBUG_ASSERT(pTrack) {
        return; 
    }
```

Makes the code save in case of later refactoring ..